### ISSUE TITLE
Make txt file optional

### ISSUE THREAD

**Issue by hsperr** [hsperr/Tesseract-Boxeditor#1]
Right now the TXT file containing the letters is not optional and needs to be in the same folder.

**Comment by felixvd** [comment]
It's imo by far the most elegant way to assure persistency. Saving the files in a subfolder is an easy change, but it's harder to move sets of files around that way.

**Comment by hsperr** [comment]
agreed, I actually thought since tesseract format is something like

```
<character> x1 y1 x2 y2
```

we can immediately generate a 'proper' boxfile, would actually be super neat if there was a way to type the text too but maybe not with opencv

**Comment by felixvd** [comment]
1) OpenCV file I/O is YML iirc.
2) I think typing text is fastest done separately in bulk, after all boxes are drawn. Generating a different set of txt files with human-readable lines like that would probably work well.
